v2/googlecloud-to-neo4j/src/main/java/com/google/cloud/teleport/v2/neo4j/model/helpers/SqlQuerySpec.java

```diff
@@ -0,0 +1,13 @@
+package com.google.cloud.teleport.v2.neo4j.model.helpers;
+
+import lombok.Builder;
+
+/**
+ * Convenience object for invoking SQL query as well as providing descriptions for read and cast phase of transform.
+ */
+@Builder
+public class SqlQuerySpec {
+    public String readDescription;
```

Comment:
Any reason all the helper classes have public instanve variables and no getters/setters?